src/core/include/cesium/omniverse/FabricAttributesBuilder.h

```diff
@@ -1,17 +1,22 @@
 #pragma once
 
-#include <omni/fabric/SimStageWithHistory.h>
+#include <omni/fabric/AttrNameAndType.h>
+
+namespace omni::fabric {
+class StageReaderWriter;
+}
 
 namespace cesium::omniverse {
 
 class FabricAttributesBuilder {
   public:
     void addAttribute(const omni::fabric::Type& type, const omni::fabric::Token& name);
-    void createAttributes(const omni::fabric::Path& path) const;
+    void createAttributes(omni::fabric::StageReaderWriter& fabricStage, const omni::fabric::Path& path) const;
```

Comment:
Are there situations where we need to write to a specifically provided stage? I'm wondering if having to obtain and pass in the stage creates more work when creating attributes Fabric prims.

Reply:
This class should probably follow what the other classes are doing at take a `pContext` in the constructor. I'll make the change.